src/renderer/css/settings.css

```diff
@@ -32,4 +32,9 @@ body {
 
 #content {
   height: 100%;
+}
+
+
+.removeme {
+  opacity: 1;
```

Comment:
I think I used that to create a class that was suposed to be removed afterwards but forgot to do it. I think it was to prevent force appearance of something. I'll delete it